static/stable/web_help/docs/zowe_ims_update_program.html

```diff
@@ -81,7 +81,7 @@ <h4>Options</h4>
 <p><code>--option</code>  | <code>-o</code> <em>(string)</em></p>
 <ul>
 <li>
-<p>Specifies to return response lines for all resources that are processed. It is
+<p>Specifies to return response lines for all resources that are processed.  It is
```

Comment:
Not sure if I understand the meaning of this sentence correctly, but I would suggest the following change: 

```suggestion
<p>Returns response lines for all resources that are processed.  The option is only valid if `--name * (ALLRSP)` is specified.
```

If this suggestion doesn't introduce any clarity, please disregard it :)

Reply:
I don't think we want to make edits to the web help, since it is automatically generated content, but we can update help text in the [zowe/zowe-cli](https://github.com/zowe/zowe-cli) repository for suggested changes like this.